analytics/views/remote_activity.py

```diff
@@ -20,17 +20,25 @@ def get_remote_server_activity(request: HttpRequest) -> HttpResponse:
 
     query = SQL(
         """
-        with icount as (
+        with icount_id as (
             select
                 server_id,
-                max(value) as max_value,
-                max(end_time) as max_end_time
+                max(id) as max_count_id
```

Comment:
Hmm, we have an index on `["server", "property", "subgroup", "end_time"]`, but not without the `end_time` entry; is this subquery indexed well?